tgui/packages/tgui/interfaces/AirVendor.tsx

```diff
@@ -1,12 +1,13 @@
 /**
  * @file
  * @copyright 2023
- * @author Garash (https://github.com/Garash2k)
+ * @author Original Garash (https://github.com/Garash2k)
+ * @author Changes Sovexe (https://github.com/Sovexe)
  * @license ISC
  */
 
 import { useBackend } from "../backend";
-import { LabeledList, NumberInput, Button, Section, Dimmer } from "../components";
+import { Button, Dimmer, Flex, LabeledList, Section, Slider } from "../components";
```

Comment:
Ooh, alphabetical! I wonder if there's a lint rule that could validate and order them automatically on save.